Always use devmapper for kpartx

Similar to wrs-always-use-libdevmapper.patch
we always use the devmapper supplied by LVM2
We do not use the host detect because
this is a cross build.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Aws Ismail <aws.ismail@windriver.com>

diff --git a/kpartx/Makefile b/kpartx/Makefile
index 2a07334..24bdabc 100644
--- a/kpartx/Makefile
+++ b/kpartx/Makefile
@@ -4,13 +4,7 @@
 #
 include ../Makefile.inc
 
-CFLAGS += -I. -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
-
-LIBDM_API_COOKIE = $(shell grep -Ecs '^[a-z]*[[:space:]]+dm_task_set_cookie' /usr/include/libdevmapper.h)
-
-ifneq ($(strip $(LIBDM_API_COOKIE)),0)
-	CFLAGS += -DLIBDM_API_COOKIE
-endif
+CFLAGS += -I. -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -DLIBDM_API_COOKIE
 
 LDFLAGS = -ldevmapper
 OBJS = bsd.o dos.o kpartx.o solaris.o unixware.o dasd.o sun.o \
